src/Messenger.js

```diff
@@ -78,7 +78,7 @@ module.exports = class Messenger {
         if (message.type === 'text') { // eslint-disable-line
           messageToUpload = message.message;
         } else if (message.type === 'image') {
-          messageToUpload = message.message + '\n' + message.image;
+          messageToUpload = message.image;
```

Comment:
@65black What happens when there is `message.message`? 

What do you think about doing this:
```
messageToUpload = message.message 
    ? message.message + '\n' + message.image
    : message.image;
```